### PR TITLE
Documentation: Update README and improve GraphQL doku

### PR DESCRIPTION
**Pull request by Jonasdoubleyou**
This proposes some conventions for /graphql. 
Also updated the main README to cross link all the documentation, and removed some repetitive commentary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,61 +1,51 @@
...
-##### Issues
-
-While we are always happy about the contribution of issues, but our own roadmap doesn't allow us too much room to tackle requests for new features.
-
-##### Code
-
-If you want to contribute code containing fixes or new features, feel free to send us a pull request AFTER reading the information below.
-
-##### Security Issues
+We're always happy and open about contributions, please  [contact us](mailto:support@lern-fair.de) in case you are interested in joining our
```
Comment (lsc0498) [inline README.md:46]: team@lern-fair.de is the official mail address of HR 😉

**Review by lsc0498 — APPROVED** [review]
Looks good, I'm fine with it. Thanks a lot!🙏

File: README.md (hunk 1 above)
Comment (Jonasdoubleyou) [reply README.md:46]: @lsc0498 okay should I add that or should we add a contact email for Tech directly?

Comment (lsc0498) [reply README.md:46]: No, team@ is the best for this purpose I guess. All other applications and stuff like that also go there.

**Review by lsc0498 — APPROVED** [review]

**Pull request merged**